nexus/db-model/src/ip_pool.rs

```diff
@@ -34,18 +83,25 @@ pub struct IpPool {
     #[diesel(embed)]
     pub identity: IpPoolIdentity,
 
+    /// The IP version of the pool.
+    pub ip_version: IpVersion,
+
     /// Child resource generation number, for optimistic concurrency control of
     /// the contained ranges.
     pub rcgen: i64,
 }
 
 impl IpPool {
-    pub fn new(pool_identity: &external::IdentityMetadataCreateParams) -> Self {
+    pub fn new(
+        pool_identity: &external::IdentityMetadataCreateParams,
+        ip_version: IpVersion,
+    ) -> Self {
         Self {
             identity: IpPoolIdentity::new(
                 Uuid::new_v4(),
                 pool_identity.clone(),
             ),
+            ip_version,
             rcgen: 0,
         }
     }
```

Comment:
Fine as-is, but this would be cuter as `new_v4` and `new_v6`. Or maybe I'm just comfortable with `new_v4` from `uuid`.

Reply:
Sure, I can add that.

Reply:
Added in e8ff83a94. I'm not using them anywhere, but I can change the callsites if that's preferable?

Reply:
I think it's fine. It would only save a few lines here and there.